e2e/content-services/metadata/metadata-properties.e2e.ts

```diff
@@ -135,7 +135,7 @@ describe('CardView Component - properties', () => {
         metadataViewPage.clickOnInformationButton();
 
         metadataViewPage.checkMetadataGroupIsNotExpand('EXIF');
-        metadataViewPage.checkMetadataGroupIsNotExpand('properties');
+        metadataViewPage.checkMetadataGroupIsExpand('properties');
```

Comment:
@dhrn This is not a fix, change the e2e test to allow your modify to pass the test is not the right thing to do this time. 
Most probably something you changed has modified this behavior that is a wanted behavior when  click on more information EXIF and properties tab has to be NOT expanded